cpp/test/prims/batched/information_criterion.cu

```diff
@@ -61,20 +61,23 @@ struct BatchedICInputs {
 template <typename T>
 class BatchedICTest : public ::testing::TestWithParam<BatchedICInputs<T>> {
  protected:
+  BatchedICTest()
+    : params(::testing::TestWithParam<BatchedICInputs<T>>::GetParam()),
+      res_d(sizeof(T) * params.batch_size, stream)
+  {
+  }
+
   void SetUp() override
   {
     using std::vector;
-    params = ::testing::TestWithParam<BatchedICInputs<T>>::GetParam();
 
-    // Create stream and allocator
+    // Create stream
     RAFT_CUDA_TRY(cudaStreamCreate(&stream));
```

Comment:
Not something that needs to be updated in this PR, but we should be creating just about all streams in the handle.